.github/ISSUE_TEMPLATE/2-Enhancement.md

```diff
@@ -0,0 +1,5 @@
+---
+name: Enhancement
+about: You want to propose an enhancement to expresslrs. 🎉
+labels: 'enhancement'
+---
```

Comment:
```suggestion
---

<!--- Insert your Suggestion here -->


*Instead of commenting "I want this too", consider leaving a :+1: instead, to reduce spam and clutter*
```

Reply:
I added this to the template 👍